app/models/project.rb

```diff
@@ -71,6 +71,18 @@ def create_releases_for_branch?(branch)
     release_branch == branch
   end
 
+  # Whether to create a new docker image when the branch is updated.
+  #
+  # branch - The String name of the branch in question.
+  #
+  # Returns true if new releases should be created, false otherwise.
+  # If the docker_release_branch of the project wasn't set, fallback
+  # to the release_branch for backward compatibility.
+  def build_docker_image_for_branch?(branch)
+    deploy_with_docker? && auto_release_docker_image? &&
```

Comment:
`building?` checks if `auto_release` is set ... can we make the naming more consistent ?